OSDOCS-16850 - CQA for OVNK-1 part 2 (#109553)
openshift/openshift-docs: modules/nw-ovn-kubernetes-examine-sb-database-contents-ref.adoc

[id="nw-ovn-kubernetes-examine-sb-database-contents-ref_{context}"]
= Command-line arguments for ovn-sbctl to examine southbound database contents

[role="_abstract"]

> The following table describes the command-line arguments that can be used with `ovn-sbctl` to examine the contents of the southbound database.

Reviewer: etc (up to you)
```suggestion
The following table describes the command-line arguments that you can use with `ovn-sbctl` to examine the contents of the southbound database.
```
